internal/cache/controller_obj_cache.go

```diff
@@ -2006,8 +2062,11 @@ func (c *AviObjCache) AviObjVSCachePopulate(client *clients.AviClient, cloud str
 								c.PopulateHttpPolicySetToCache(client, cloud)
```

Comment:
@arihantg: If we look at `PopulateHttpPolicySetToCache` function, it fetches all httpps which is created by AKO user and populate the cache. And then we retrieve out httpps by uuid at line number 2063. I think fetching all httpps each time is not required here (at line number 2062) as we are going through httpps attached to each VS. 

Instead of this call, we can call your function `AviPopulateHttpPolicySetbyUUID` here. So that it will fetch it from uuid.

All HTTPPS cache update is already happened as part of `AviRefreshObjectCache`

Reply:
I have replaced the 2nd call to PopulateHttpPolicySetToCache with PopulateHttpPolicySetToCache.